# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/svgfe-subregion-bigger.yaml
# Commit: c0f58fd74e56
# Full Hash: c0f58fd74e56b24f67fccae08097d151e21a0f80
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-09 16:15:25
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r 3e2ec3a3b738 -r c0f58fd74e56 gfx/wr/wrench/reftests/filters/svgfe-subregion-bigger.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/svgfe-subregion-bigger.yaml	Thu May 09 04:09:00 2024 +0000
@@ -0,0 +1,35 @@
+---
+root:
+  items:
+    - type: stacking-context
+      bounds: [0, 50, 800, 800]
+      filters:
+        - svgfe: SourceGraphic
+        - svgfe: SourceAlpha
+        - svgfe: flood
+          linear: false
+          # this is expected to be at 10,60
+          subregion: [10, 10, 80, 100]
+          color: [255, 0, 0, 0.7]
+        - svgfe: flood
+          linear: false
+          # this is expected to be at 50,60
+          subregion: [50, 10, 80, 100]
+          color: [0, 255, 0, 0.7]
+        - svgfe: blendnormal
+          linear: false
+          # this is expected to be at 10,60
+          subregion: [10, 10, 160, 100]
+          in: 3
+          in2: 2
+        - svgfe: blendnormal
+          linear: false
+          # this is expected to be at 10,60
+          subregion: [10, 10, 160, 100]
+          in: 0
+          in2: 4
+      items:
+        - type: rect
+          # this is expected to be at 90,60
+          bounds: [90, 10, 80, 100]
+          color: [0, 0, 255, 0.7]